app/models/thing.rb

```diff
@@ -11,6 +11,7 @@ class Thing < ActiveRecord::Base
   validates :city_id, uniqueness: true, allow_nil: true
   validates :lat, presence: true
   validates :lng, presence: true
+  attr_accessor :owned_by_you
```

Comment:
It feels a little weird to have this attribute exist here, but only be populated by the controller. This'll probably be mitigated by moving the method for checking the owner into here.